UI/window-basic-main.cpp

```diff
@@ -11053,6 +11053,33 @@ QColor OBSBasic::GetSelectionColor() const
 	}
 }
 
+const std::vector<MultitrackVideoViewInfo> &
+OBSBasic::GetAdditionalMultitrackVideoViews()
+{
+	return multitrackVideoViews;
+}
+
+void OBSBasic::MultitrackVideoRegister(const char *name,
```

Comment:
It seems like the register and unregister code could live in api-interface.cpp. There's no real benefit having functions here if only api-interface calls them.

Reply:
right, and api-interface already has access to window basic main internals

Reply:
Looks like this feedback was addressed.